bonsai/Appearance/Appearance.swift

```diff
@@ -8,6 +8,19 @@
 import Foundation
 import SwiftUI
 
+enum BonsaiFont {
+    static let bonsai_34 = UIFont.systemFont(ofSize: 34, weight: .init(rawValue: 700))
+    static let bonsai_28 = UIFont.systemFont(ofSize: 28, weight: .init(rawValue: 700))
+    static let bonsai_22 = UIFont.systemFont(ofSize: 22, weight: .init(rawValue: 700))
```

Comment:
naming is not very clear, could we name these better?
I like how `body_17` and others sound

Reply:
maybe main_34, main_28 ?

Reply:
What I meant is that `body`, `title`, `caption` etc, they are expressive, while `main` or `bonsai` doesn't give any understanding what is this font used for